src/tasks/index.ts

```diff
@@ -272,127 +272,84 @@ export namespace TasksModel {
         isArchived: boolean;
     }
 
-    export interface CreateTaskEnterpriseRequest {
-        type?: Type;
-        workflowStepId?: number;
+    interface CreateTaskBase {
         title: string;
         languageId: string;
         fileIds: number[];
-        status?: Status;
+        status?: RequestStatus;
         description?: string;
+        labelIds?: number[];
+        excludeLabelIds?: number[];
+        dateFrom?: string;
+        dateTo?: string;
+    }
+
+    export interface CreateTaskEnterpriseRequest extends CreateTaskBase {
+        type?: Type;
+        workflowStepId?: number;
         splitFiles?: boolean;
```

Comment:
this should be marked as `deprecated` and also `splitContent` needs to be added here

https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.tasks.post

Reply:
or actually you can keep it as is. I have these changes in my PR #328 . Will get merge yours on top